scripts_8.0/utils.py

```diff
@@ -281,6 +282,18 @@ def wait_for_mysql():
         logdbg('mysql server is ready')
         return
 
+def wait_for_memcached():
+    while True:
+        try:
+            with telnetlib.Telnet(host='memcached', port=11211, timeout=3) as tn:
```

Comment:
Hello! 
I came here to have a look at the memcached configuration options.
As far as I see it, there is no MEMCACHED_HOST environment variable, but I guess one day there might be one.

Making this MR future-proof, one could already now add it here.

As the default value would be used when no configuration is provided, I only see this as saving some work when adding the memcached config later.

```python
host=get_conf('MEMCACHED_HOST', 'memcached')
```